CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 - Automatically configure Intellisense to use compile_commands.json generated
   by Meson.
+- Meson `tasks.json` entries specifying `target` will now need to include the target's type. E.g. `"target": "relative/path/to/targetname:shared_library"`.
```

Comment:
I admit I am not an expert of tasks.json, but I hate magic strings like this.

Can we do something like the following?

`
{
  "target": { "path": "rel/path", "type" : "the type which is an enum" }
}

Reply:
IMHO since it's just the meson command line target name parameter, but with a mandatory type, personally I prefer it as the single string even though I agree with your principle. Plus there would be changes to the task generation and node commands presumably.

Sorry, not mine it was from c5a5166, just thought I'd add the changelog note whilst doing related stuff as it took a while to figure out why my tasks were broken...